comps/comps-foreman-rhel7.xml

```diff
@@ -306,18 +306,14 @@
         Do not edit this section manually and use ./comps_doc.sh script
       -->
       <!--RGD-START-->
-      <packagereq type="default">nodejs-diff-doc</packagereq>
       <packagereq type="default">nodejs-expose-loader-doc</packagereq>
       <packagereq type="default">nodejs-file-loader-doc</packagereq>
       <packagereq type="default">nodejs-ipaddr.js-doc</packagereq>
       <packagereq type="default">nodejs-jquery.cookie-doc</packagereq>
       <packagereq type="default">nodejs-jquery-doc</packagereq>
       <packagereq type="default">nodejs-jquery-flot-doc</packagereq>
-      <packagereq type="default">nodejs-jquery-ujs-doc</packagereq>
-      <packagereq type="default">nodejs-jstz-doc</packagereq>
       <packagereq type="default">nodejs-multiselect-doc</packagereq>
       <packagereq type="default">nodejs-select2-doc</packagereq>
-      <packagereq type="default">nodejs-url-loader-doc</packagereq>
```

Comment:
This nodejs comps changes don't seem related, @dgoetz can you comment on these or drop them from the diff to merge?

Reply:
I think they're already in rpm/develop so a trivial rebase should be sufficient.